cuegui/cuegui/JobMonitorTree.py

```diff
@@ -259,6 +259,15 @@ def removeFinishedItems(self):
         for item in self.findItems("Finished", QtCore.Qt.MatchFixedString, COLUMN_STATE):
             self.removeItem(item)
 
+    def removeAllJobsExcept(self, keepList):
+        """Removes jobs where the name doesn't match the given search string"""
```

Comment:
Can we add a docstring param for keepList?

Reply:
Done.